src/app/shared/issue/label/label.component.css

```diff
@@ -22,8 +22,8 @@
 }
 
 .labelInfo {
-  display: grid;
-  grid-template-columns: 20% auto;
+  display: flex;
+  font-size: 1em;
   align-items: center;
   text-align: left;
   grid-gap: 20%;
```

Comment:
You may remove this grid attribute if grid still does not work

Reply:
Using the grid template with a fixed size: `20em auto 20em`, the dropdown still appears like this on safari:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/67156011/179391355-2a0c6b63-68e1-4289-8d10-714e6692512d.png">

Even after removing `grid-gap`, the dropdown still looks the same as above

Reply:
Yes, this is because the nesting of the divs make it 2 elements. 

<img width="352" alt="image" src="https://user-images.githubusercontent.com/9072299/179398006-b754e416-efc5-43df-9deb-b5657ea7bdca.png">

Sorry for the unclear instructions,
try:
```
.labelInfo {
grid-template-columns: 1fr 2fr;
}
.labelLine {
grid-template-columns: 3fr 1fr;
}
```
If you believe that `flex` is a better approach, change the `.labelLine` css to not use `auto`. That's probably the root of the issue.
Alternatively you may also want to deconstruct the buttons into a flat hierarchy, so that a grid with 3 columns work outright.

Reply:
Ah, I see, I tried out:

```
.labelInfo {
grid-template-columns: 1fr 2fr;
}
.labelLine {
grid-template-columns: 3fr 1fr;
}
```

However, it still did not fix the issue on safari for me, but somehow removing the grid styling did:


https://user-images.githubusercontent.com/67156011/179401313-ee4e8aa3-ee37-486f-a647-43b7df04762a.mov

I think it could be that i'm using version 13.1 of safari and hence this issue is caused due to the outdated version, if its ok I could revert the changes and keep only the fontsize change

Reply:
Thanks for verifying and helping with the investigations!
Let's remove the lines as per your video, and keeping the changes on `.labelLine`. After which the PR will be ready to merge :)

Reply:
Alright, I've made the changes! Thank you for the review